scripts/fetch-ui-assets.sh

```diff
@@ -26,9 +26,16 @@ curl -Ls https://github.com/influxdata/ui/releases/download/OSS-Master/sha256.tx
 # Download the tar file containing the built UI assets.
 curl -L https://github.com/influxdata/ui/releases/download/OSS-Master/build.tar.gz --output build.tar.gz
 
+
 # Verify the checksums match; exit if they don't.
-echo "$(cat sha256.txt)" | sha256sum --check -- \
-    || { echo "Checksums did not match for downloaded UI assets!"; exit 1; }
+if [ "$(uname)" = "FreeBSD" ]; then
```

Comment:
thank you -- i really appreciate you driving this home!  this is purely a suggestion and i don't feel strongly about this:

what do you think about using a `case` block for detecting the operating system?  something like:

```
case $(uname -s) in
   FreeBSD)
    ...
   Darwin)
    ...
   Linux)
    ...
    *)
      ...
esac

Reply:
@ayang64 I will be happy to refactor the code :) But while no maintainer interacts with the pull request, I don't want to lose my time on this...

Reply:
👍 to @ayang64's suggestion. `sha256sum` is non-standard on Mac as well, so I'd suggest:
```
case $(uname -s) in
    FreeBSD|Darwin)
        # Use shasum --algorithm 256
    *)
        # Use sha256sum
esac
```